orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pollers/RestorePinnedServerGroupsPoller.java

```diff
@@ -76,13 +78,35 @@ public RestorePinnedServerGroupsPoller(@Qualifier("jedisPool") Pool<Jedis> jedis
                                          Registry registry,
                                          ExecutionLauncher executionLauncher,
                                          ExecutionRepository executionRepository) {
+    this(
+      jedisPool,
+      objectMapper,
+      oortService,
+      retrySupport,
+      registry,
+      executionLauncher,
+      executionRepository,
+      new PollerSupport(objectMapper, retrySupport, oortService)
+    );
+  }
+
+  @VisibleForTesting
+  RestorePinnedServerGroupsPoller(@Qualifier("jedisPool") Pool<Jedis> jedisPool,
```

Comment:
Logic is unchanged ... just slightly refactored to account for the extracted classes